ztp/source-crs/ReduceMonitoringFootprint.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    grafana:
+      enabled: false
+---
```

Comment:
PolicyGen requires there to be a single CR per source file do make updates. We should split this into two files to allow the user to override if/as desired.